app/assets/stylesheets/frontend/views/_html-publications.scss

```diff
@@ -99,6 +99,16 @@
         padding-left: 0;
         li {
           list-style: none;
+          &.numbered {
+            margin-left: $gutter;
+            a {
+              .heading-number {
+                @include bold-19($tabular-numbers: true);
+                position: absolute;
+                right: 92%;
```

Comment:
@russellthorn Is this going to hold up at varying screen widths? In my experience % based absolute positioning tends to be quite brittle :(